res/phantom.js

```diff
@@ -1,6 +1,7 @@
 /* global phantom */
 var webpage = require( 'webpage' ),
-	currentUrl = phantom.args[ 0 ],
+	system = require( 'system' ),
+	currentUrl = system.args[ 2 ],
```

Comment:
Perhaps logging should be introduced, but I think that we should introduce features and bug fixes separately :)